Dragonflight/WarriorArms.lua

```diff
@@ -428,6 +437,11 @@ spec:RegisterStateExpr( "glory_rage", function ()
     return gloryRage
 end )
 
+local TriggerMeleeCriticalHit = setfenv( function() -- Seasoned Soldier: Auto attack crits generate 20% more rage.
```

Comment:
I don't think we need to do this, unless we know a crit is going to happen in the forecasted game state (i.e., you have 100% crit or a buff that will guarantee that your next melee will be a crit).  I don't think that's the case.

Reply:
Removed melee crits within https://github.com/Hekili/hekili/pull/1670/commits/2517d7b4869af49fa60079d1521a38b4833c9803 as they can't be forecasted as you say.